src/tools/perf/api/libperf.h

```diff
@@ -98,6 +98,60 @@ enum {
 };
 
 
+/*
+ *   +------+                                                        +------+
+ *   |HOST 0|----------------------PEER_TX(AM)---------------------->|HOST 1|
+ *   +------+        +---------+                  +---------+        +------+
+ *      |            |DPU DMN 0|                  |DPU DMN 1|            |
+ *      |            +---------+                  +---------+            |
+ *      |                 |                            |                 |
+ *      |---HOST_INIT---->|                            |<----HOST_INIT---|
+ *      |                 |---------PEER_INIT--------->|                 |
+ *      |                 |                            |                 |
+ *      |---HOST_SEND---->|                            |<--HOST_RECV(T)--|
+ *      |                 |--------PEER_TX(AM)-------->|                 |
+ *      |<--HOST_SEND_ACK-|                            |-HOST_RECV_ACK-->|
+ *      |                 |                            |                 |
+ *      |----HOST_FIN---->|                            |<----HOST_FIN----|
+ *
+ * (AM) used only when AM communication is configured
+ * (T) used only when tag matching communication is configured
+ */
+#define UCP_FOREACH_PERF_DAEMON_AM_ID(X) \
+    /* Host->DPU daemon AM initialization message */ \
+    X(UCP_PERF_DAEMON_AM_ID_HOST_INIT,     0) \
+    /* Host->DPU daemon AM to initiate data transfer to the remote peer */ \
+    X(UCP_PERF_DAEMON_AM_ID_HOST_SEND,     1) \
+    /* DPU daemon->Host AM to ack the data tx operation initiated by HOST_SEND */ \
+    X(UCP_PERF_DAEMON_AM_ID_HOST_SEND_ACK, 2) \
+    /* Host->DPU daemon, (optional) AM to initiate data receive. Not used when \
+     * DPU peers communicate by AM. Will be used by tag matching */ \
+    X(UCP_PERF_DAEMON_AM_ID_HOST_RECV,     3) \
+    /* DPU daemon->Host AM to ack the receive completion. This message is sent \
+     * unsolicited in case of AM communication. For tag matching transport it \
+     * confirms the completion of HOST_RECV request */ \
+    X(UCP_PERF_DAEMON_AM_ID_HOST_RECV_ACK, 4) \
+    /* Host->DPU daemon AM finalize message */ \
+    X(UCP_PERF_DAEMON_AM_ID_HOST_FIN,      5) \
```

Comment:
maybe:

```suggestion
    X(UCP_PERF_DAEMON_AM_ID_DAEMON_FIN,      5) \
```

Reply:
Well, since you proposed to exclude HOST from enum IDs, I would keep maybe `UCP_PERF_DAEMON_AM_ID_FIN`?
Otherwise DAEMON appears twice in a single ID